src/help/clear.js

```diff
@@ -0,0 +1,11 @@
+module.exports = `
+Usage: clear
+Clear the terminal screen
+
+      --help        display this help and exit
+
+Clear ignores any command-line parameters that may be present.
+
+Report pwd bugs to <https://github.com/dthree/cash>
```

Comment:
Help looks good!

Just replace `Report pwd bugs to` with `Report clear bugs to`.